docs/UNRAID_API_OPERATIONS.md

```
@@ -0,0 +1,290 @@
+# Unraid GraphQL API Operations
+
+Generated via live introspection at `2026-02-15 23:45:50Z`.
+
+## Schema Summary
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)